### PR TITLE
Story/10760 2 g plus check in

### PR DESCRIPTION
**Pull request by nickguendling**
## Description
Story pull request

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-10760

### CODE REVIEW

**Review by naveeddotio — APPROVED** [review]

**Review by marcussc — APPROVED** [review]

**Review by 30mar — APPROVED** [review]

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings [hunk 1]
```diff
@@ -2902,6 +2890,44 @@ Die Angaben werden statistisch ausgewertet, sie werden nicht zu einem Profil ges
...
+
+"HealthCertifiedPerson_AdmissionState_FAQ_Placeholder" = "FAQ";
+
+"HealthCertifiedPerson_AdmissionState_shortTitle_title2G" = "2G";
+
+"HealthCertifiedPerson_AdmissionState_shortTitle_title2GPlus" = "2G+";
+
+"HealthCertifiedPerson_AdmissionState_shortTitle_title3G" = "3G";
+
+"HealthCertifiedPerson_AdmissionState_shortTitle_title3GPlus" = "3G+";
```
Comment (CV113) [inline src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings:2927]: @KathrinLuecke Should we replace here + with PLUS ?

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings [hunk 2]
```diff
@@ -2902,6 +2890,44 @@ Die Angaben werden statistisch ausgewertet, sie werden nicht zu einem Profil ges
...
+
+"HealthCertifiedPerson_AdmissionState_description3GPlus" = "Ihre Zertifikate erfüllen die 3G-Plus-Regel. Wenn Sie Ihren aktuellen Status vorweisen müssen, schließen sie diese Ansicht und zeigen Sie den QR-Code auf der Zertifikatsübersicht.";
+
+"HealthCertifiedPerson_AdmissionStateFAQ" = "Mehr Informationen finden Sie in den %@.";
+
+"HealthCertifiedPerson_AdmissionState_FAQ_Placeholder" = "FAQ";
+
+"HealthCertifiedPerson_AdmissionState_shortTitle_title2G" = "2G";
+
+"HealthCertifiedPerson_AdmissionState_shortTitle_title2GPlus" = "2G+";
```
Comment (CV113) [inline src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings:2923]: Should we replace here + with PLUS ?

Comment (CV113) [reply src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings:2923]: @KathrinLuecke

**Review by CV113 — APPROVED** [review]
UA approved

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings (hunk 1 above)
Comment (KathrinLuecke) [reply src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings:2927]: I guess these are just the short labels which appear then on the blue button, so I would say we keep it here as it is (and only have 2G-Plus in the Fliesstext)

**Pull request merged**